src/pages/[platform]/build-a-backend/storage/upload/index.mdx

```diff
@@ -244,7 +244,7 @@ const uploadDataInBrowser = async () => {
 
 ```html
 <h2>Upload File</h2>
-<input type="file" onChange="{()" ="" /> uploadDataInBrowser(event)} />
+<input type="file" onChange={uploadDataInBrowser} />
```

Comment:
Is this supposed to be React code or plain HTML?

Reply:
This is react code, and is used presented as html, would make more sense to make it pure html?

Reply:
OI suggest we remove this all together.